docs/jazzy.yml

```diff
@@ -0,0 +1,16 @@
+module: MapboxDirections.swift
```

Comment:
The module name is MapboxDirections. It isn’t possible to have a `.` in the module name (one reason I’d like to rename the library).